features/supplier/view_and_edit_g_cloud_services.feature

```diff
@@ -9,13 +9,7 @@ Background:
   And we accept that suppliers application to the framework
   And that supplier returns a signed framework agreement for the framework
   And that supplier has a service on the cloud-support lot
-  When I visit the /suppliers page
-  # The following step only works by virtue of there only being a single service for this supplier - multiple services on
-  # multiple frameworks will cause multiple "View services" links to be present
-  And I click 'View services'
-  Then I am on the 'G-Cloud 10 services' page
-  When I click 'Test cloud support service'
-  Then I am on the 'Test cloud support service' page
+  Then that supplier visits their page for that service
```

Comment:
A couple of things:

 - This does end up testing _much_ less in magically jumping straight to the target page. We usually try to limit the number of assumptions of urls we jump to and instead navigate to them "naturally".
 - `Then` steps are generally intended to be used with assertions. This step doesn't really assert anything other than getting a 200 for a request to the target page, does it?